packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -404,19 +404,19 @@ export const guides = [
             title: "Okta Sign-In Widget upgrade",
             subLinks: [
               {
-                title: "Upgrade your widget",
+                title: "Upgrade your Widget",
```

Comment:
we don't capitalize when referring to widget any longer. That's a more recent change - you may have missed the discussion. 

```suggestion
                title: "Upgrade your widget",
```